PWGHF/TableProducer/treeCreatorOmegacSt.cxx

```diff
@@ -508,7 +557,9 @@ struct HfTreeCreatorOmegacSt {
           if (((std::abs(bachelor.tpcNSigmaKa()) < maxNSigmaBachelor) || (std::abs(bachelor.tpcNSigmaPi()) < maxNSigmaBachelor)) &&
               (std::abs(v0TrackPr.tpcNSigmaPr()) < maxNSigmaV0Pr) &&
               (std::abs(v0TrackPi.tpcNSigmaPi()) < maxNSigmaV0Pi)) {
-            std::array<double, 2> massesOmegacDaughters{o2::constants::physics::MassOmegaMinus, o2::constants::physics::MassPiPlus};
+
+            std::array<double, 2> massesOmegacToOmegaPi{o2::constants::physics::MassOmegaMinus, o2::constants::physics::MassPiPlus};
+            std::array<double, 2> massesOmegacToOmegaK{o2::constants::physics::MassOmegaMinus, o2::constants::physics::MassKPlus};
             std::array<double, 2> massesXicDaughters{o2::constants::physics::MassXiMinus, o2::constants::physics::MassPiPlus};
```

Comment:
Use `NProngs` or `NDaughters` wherever you create containers with elements related to the decay products.

Reply:
The code is more understandable when the meaning is conveyed through the names of the entities. This is even more relevant in your case since you operate with 6 different arrays of masses which happen to all have 2 prongs.

Reply:
Please address it in the next PR.